subsys/bluetooth/controller/ll_sw/lll.h

```diff
@@ -185,26 +196,13 @@ struct node_rx_hdr {
 
 	enum node_rx_type   type;
 	u16_t               handle;
-};
 
-/* Footer of node_rx_pdu.
- * TODO: Eliminate footer (move contents to header) to avoid pointer arithmetic
- */
-struct node_rx_ftr {
-	void  *param;
-	void  *extra;
-	u32_t ticks_anchor;
-	u32_t us_radio_end;
-	u32_t us_radio_rdy;
+	struct node_rx_ftr  rx_ftr;
```

Comment:
It would make sense to rename the footer into something like metadata.

Reply:
In this PR it was done this way to keep the changes to the rest of the code base relatively simple (i.e. node_rx_ftr struct was not changed).
Renaming should IMHO be done if/when the struct is renamed.